tests/Elixir114.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Tests;
+
+// Runtime: elixir-1.14
+// PHP class: Elixir114
+// Entrypoint: tests.ex
+
+class Elixir114 extends Base
```

Comment:
To make Elixir test run on GitHub, we need to add it to `.travis.yml`. Could you please do that? Should be copy&paste of another runtime, and adjust.